catroid/src/main/java/org/catrobat/catroid/camera/CameraManager.java

```diff
@@ -310,6 +310,20 @@ public boolean hasCurrentCameraFlash() {
 		return currentCameraInformation.flashAvailable;
 	}
 
+	public boolean hasBackCameraFlash() {
+		if (hasBackCamera()) {
```

Comment:
you can use "early return" here:

```
return hasBackCamera() && backCameraInformation.flashAvailable:
```

Java evaluates the right hand side of the condition first, so even no null pointer exception can occur here.